adapter/models.go

```diff
@@ -87,12 +87,47 @@ func ScriptToAPI(script *model.ScriptExecution) *model.ScriptExecution {
 	return script
 }
 
+func ScriptsToAPI(scripts []*model.ScriptExecution) []*model.ScriptExecution {
+	for i, a := range scripts {
+		scripts[i] = ScriptToAPI(a)
+	}
+	return scripts
+}
+
+func ContractToAPI(contract *model.ContractDeployment) *model.ContractDeployment {
+	contract.Address = addressToAPI(contract.Address)
+	contract.Script = contentAddressToAPI(contract.Script)
+
+	for i, e := range contract.Errors {
+		contract.Errors[i].Message = contentAddressToAPI(e.Message)
+	}
+
+	for i, a := range contract.Events {
```

Comment:
I would extract this into a method called `EventsToAPI`.

Reply:
Done